deploy/Dockerfile

```diff
@@ -5,19 +5,25 @@ ENTRYPOINT /usr/bin/go
 
 FROM local_go as builder
 
+ENV GOMODCACHE=/.cache/gomod-cache
```

Comment:
Updated Dockerfile with caches for faster local build.  Build time reduced from >60seconds to now under 2-5 seconds after the initial build.